rest_framework/fields.py

```diff
@@ -467,7 +467,7 @@ def get_attribute(self, instance):
                 )
             )
             raise type(exc)(msg)
-        except (KeyError, AttributeError) as exc:
+        except (AttributeError, KeyError) as exc:
```

Comment:
what benefit does this change provide?

Reply:
the benefit is that exception are in alphabetical order